lib/pip.js

```diff
@@ -210,7 +219,7 @@ function installRequirements(targetFolder, serverless, options) {
           ])
         );
       }
-      pipCmd = ['/bin/bash', '-c', '"' + commands.join(' && ') + '"'];
+      pipCmd = ['/bin/bash', '-c', '"' + commands.join(' ; ') + '"'];
```

Comment:
What's the motivation for changing this? Would you really want to keep running commands if one of them fails?